pom.xml

```diff
@@ -114,11 +113,37 @@
             <version>2.7.2</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>git</artifactId>
+           <version>3.9.1</version>
+        </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>jackson2-api</artifactId>
             <version>2.7.3</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>credentials</artifactId>
+            <version>2.1.14</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>junit</artifactId>
+            <version>1.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci</groupId>
+            <artifactId>annotation-indexer</artifactId>
+            <version>1.12</version>
+        </dependency>
+        <!--<dependency>-->
+            <!--<groupId>org.jenkins-ci.main</groupId>-->
+            <!--<artifactId>jenkins-test-harness-htmlunit</artifactId>-->
+            <!--<version>2.18-1</version>-->
+            <!--<scope>compile</scope>-->
+        <!--</dependency>-->
```

Comment:
@gautamabhishek46 When you comment out code, use a "TODO" comment that indicates the future intent. Otherwise we do not know if the code needs to be removed, or added back.

Reply:
Ok